src/kata5.reachDestination.js

```diff
@@ -1,3 +1,8 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+    let journeyTime = distance / speed; 
```

Comment:
You could declare this variable as a const instead of a let as you are never overwriting the value stored in it.